tests/test_minpvprocessor.cpp

```diff
@@ -67,9 +67,9 @@ BOOST_AUTO_TEST_CASE(Pinch)
     pinch_no_gap = true;
     minpv_result = mp1.process(thickness, z_threshold, pv, minpvv, actnum, fill_removed_cells, z1.data(), pinch_no_gap);
     BOOST_CHECK_EQUAL_COLLECTIONS(z1.begin(), z1.end(), zcornAfter.begin(), zcornAfter.end());
-    BOOST_CHECK_EQUAL(minpv_result.nnc.size(), 1);
+    BOOST_CHECK_EQUAL(minpv_result.nnc.size(), 0); // No nnc because deactivated cell is to broad
```

Comment:
Too.

Reply:
Thanks. Still learning. A bit embarrassing that I have been doing this mistake all my live.

Reply:
In the same vein, what is "broad" here?  Thick?  Wide?  Long?

Reply:
reworded for better clarity (hopefully)